paymentapi/src/main/resources/application-local.properties

```diff
@@ -10,6 +10,8 @@ payment.card_token.encryption.password=testest
 
 payment_default_notify_url=http://localhost:8085/v1/payment/onlinePayment/notify
 payment_default_return_url=http://localhost:8085/v1/payment/onlinePayment/return
+
+serviceconfiguration.url=http://localhost:8187/serviceconfiguration/
```

Comment:
Could you use add this to [here](https://github.com/City-of-Helsinki/verkkokauppa-core/blob/07be80a2d8914bc31bbd5f915303788cbb13f70c/common/src/main/java/fi/hel/verkkokauppa/common/configuration/ServiceUrls.java)